BaseFilter/sections/specific.txt

```diff
@@ -588,6 +588,9 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+tickle.porn#%#//scriptlet('prevent-element-src-loading', 'iframe' , '/player/html.php?aid=pre_roll')
```

Comment:
@Alex-302 @ntnguyen1234 isn't that how it should be? Fixed.
```adblock
tickle.porn#%#//scriptlet('prevent-element-src-loading', 'iframe', '/player/html.php?aid=pre_roll')
```

Reply:
you are right